druid/src/widget/image.rs

```diff
@@ -25,7 +25,52 @@ use crate::{
     PaintCtx, Rect, RenderContext, Size, UpdateCtx, Widget,
 };
 
-/// A widget that renders an Image
+/// A widget that renders a bitmap Image.
+///
+/// Contains data about how to fill given space and interpolate pixels.
+/// Configuration options are provided via the builder pattern.
+///
+/// Note: interpolation can lead to blurry or pixelated images and so is not
+/// recommended for things like icons. Instead consider using
```

Comment:
Can lead to when? With DPI scaling? Should mention just briefly that when scaling.

Reply:
i'm not sure exactly how DPI scaling is relevant here? like images may look fine on a normal DPI screen but blurry when in hiDPI situations?